Libraries/Image/assetPathUtils.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict
+ */
+
+'use strict';
+
+import type {PackagerAsset} from './AssetRegistry';
+
+/**
+ * FIXME: using number to represent discrete scale numbers is fragile in essence because of
+ * floating point numbers imprecision.
+ */
+function getAndroidAssetSuffix(scale: number): string {
```

Comment:
This file is now duplicated between React Native and React Native CLI. Shall we consider making it a separate package, e.g. `react-native-cli-path-utils` and publish to `npm`?

CC: @TheSavior

Reply:
I feel that to keep RN and the CLI separated but having them both using these utils the separate package option is a valid solution

Reply:
Yeah. For now, I think we can ship *the copy* and figure out this as a next PR. Problem with this one is that I need to update `local-cli` folder every time something changes which is very hard. Hopefully, we can get this reviewed ASAP.

Reply:
Note that this file is for more than just images, so it does need to be in RN, to be used by other asset types.